frontend/components/MainContent/_styles.scss

```diff
@@ -10,4 +10,11 @@
   > :not(.main-content--animation-disabled) {
     animation: fade-in 250ms ease-out;
   }
+
+  &__warning-banner {
+    margin-bottom: $pad-large;
+    // TODO: figure out if sticky styling is needed?
+    position: sticky; // Needed for settings page scroll
+    z-index: 4; // Needed for settings page scroll
```

Comment:
@RachelElysia I was updating some styles around the warning banners we place on top of the page and I wasn't sure if I could delete these sticky styles or not. I didn't see them have any effect when removed but wanted to ask you first before deleting them.